### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by SadeSiddiqui**
Added a folder structure section to the readme to explain the purpose of all the folders used in the repository

### CODE REVIEW

**Comment by Johnballboa** [comment]
[Deploy Jekyll site to Pages preview environment](https://github.com/github/opensource.guide/actions/workflows/jekyll-preview.yml)

File: README.md
Comment (Johnballboa) [inline README.md:1]: README.md

**Comment by Thetzx20** [comment]
[Uploading autofile.pdf…]()

**Comment by jmeridth** [comment]
@SadeSiddiqui I believe this introduces additional maintenance on the README file when folders change. I'm not a fan. Going to ask other maintainers for feedback.

**Comment by Thetzx20** [comment]
I sincerely apologize for the inconvenience Ive had problems with my old
phone and network connection.didnt mean to cause conflict.

File: README.md
Comment (Areyyey) [reply README.md:1]: CONTRIBUTING.md

**Comment by ahpook** [comment]
+1 to what @jmeridth said - writing should help the audience and i don't feel like anyone is confused about the purpose of the `docs` subfolder...

**Pull request closed without merging**